lib/globals.css

```diff
@@ -109,7 +114,7 @@
 .tiptap ul,
 .tiptap ol {
   padding: 0 2.5rem;
-  color: #6b6f76;
+  color: var(--text-primary);
```

Comment:
So, is this intentional as well? the pr description only suggests it should be for the **p** tag?